README.md

```diff
@@ -1,20 +1,50 @@
 # CAStore
 
-**TODO: Add description**
+[![Build Status](https://travis-ci.com/ericmj/castore.svg?branch=master)](https://travis-ci.com/ericmj/castore)
+
+Up-to-date CA certificate store.
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `castore` to your list of dependencies in `mix.exs`:
+In your `mix.exs`:
 
 ```elixir
 def deps do
   [
-    {:castore, "~> 0.1.0"}
+    {:castore, ">= 0.0.0"}
```

Comment:
Let's keep the old requirement, we may still have breaking changes.

Reply:
Sounds good! Pushed a fix.